java-client-model/src/main/java/com/cumulocity/rest/representation/audit/ChangeRepresentation.java

```diff
@@ -0,0 +1,80 @@
+package com.cumulocity.rest.representation.audit;
+
+import com.cumulocity.model.audit.AuditChangeValueConverter;
+import com.cumulocity.rest.representation.AbstractExtensibleRepresentation;
+import lombok.*;
+import org.svenson.JSONProperty;
+import org.svenson.converter.JSONConverter;
+
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+@AllArgsConstructor
+public class ChangeRepresentation extends AbstractExtensibleRepresentation {
```

Comment:
is it needed in this module?

Reply:
Yes, used in `AuditRecordRepresentation` and transitively in `AuditRecordApi`